charts/garden-project/charts/project-rbac/templates/rolebinding-project-members.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: {{ include "rbacversion" . }}
+kind: RoleBinding
+metadata:
+  name: garden-project-members
+  namespace: {{ .Release.Namespace }}
+  labels:
+    garden.sapcloud.io/role: members
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: garden.sapcloud.io:system:project-member
```

Comment:
This role needs to be also reconciled by the gardener as it might not be deployed by the end-users

Reply:
Agreed, this `ClusterRole` is currently (unchanged to as it was for the past year) part of the Gardener Helm chart and not reconciled at all. In fact, only the administrator/provider of Gardener must install it once at the beginning and end-users do not need to care.

For now I would keep it as it is (`ClusterRole` creation during installation/deployment of Gardener per Helm chart) as we are thinking about introducing dedicated `Member` resources (or making it part of the `Project` resource who is member and who is not) as well (for the same reasons why we want the `Project` resource: adding/removing members from a project requires too implicit/unreliable knowledge as a special `RoleBinding` in the project's namespace having a certain name must be changed).